ddtrace/llmobs/_experiment.py

```diff
@@ -153,6 +153,12 @@ def delete(self, index: int) -> None:
         self._deleted_record_ids.append(record_id)
         del self._records[index]
 
+    @property
+    def url(self) -> str:
+        # FIXME: need to use the user's site
+        # also will not work for subdomain orgs
+        return f"https://app.datadoghq.com/llm/datasets/{self._id}"
```

Comment:
Can we get them from `self._llmobs_instance._dne_client._site`?

Reply:
not the 3 levels deep of private field access 😭

Reply:
but "yes" we'd likely follow up with that or decide to propagate the site down to the instance